src/entities/StartOptions.ts

```diff
@@ -1,8 +1,10 @@
 import { Account } from './Account';
+import { ConnectStyles } from './ConnectStyles';
```

Comment:
Maybe we can rename this type to
```suggestion
import { HubStyles } from './HubStyles';
```